README.md

```diff
@@ -221,6 +221,8 @@ If you are the maintainer of a project, you can add our badge by copying the cod
 
 ## <a name="U"> </a>U
 
+- **[Ultra-Api]([https://github.com/daoodaba975/taaru](https://github.com/bambadiagne/ultra-api))** - This is a optimized RESTful API. It's built with Flask,PostgresSQL,Redis and Docker.The aim is to show the different concepts of a backend(caching,security,...). by **[@bambadiagne](https://github.com/bambadiagne)**
```

Comment:
Duplicated project link !
need to remove `[https://github.com/daoodaba975/taaru]`

Reply:
@bambadiagne

Reply:
it's done @daoodaba975

Reply:
Great, made some fix also!